source/js/viewer-core.js

```diff
@@ -1350,16 +1351,18 @@ export default class ViewerCore
      *
      * @param pageIndex
      */
-    setCurrentPage (pageIndex)
+    setCurrentPages (currentPage, visiblePages)
     {
-        if (this.viewerState.currentPageIndex !== pageIndex)
+        // these arrays will never contain objects so this should be safe
+        if (JSON.stringify(this.viewerState.currentPageIndices) !== JSON.stringify(visiblePages))
```

Comment:
What is the purpose of passing this through `stringify`?

Reply:
This seems to be the easiest way to compare simple arrays (that don't contain objects). Regular strict inequality doesn't work